javascript/ql/src/semmle/javascript/StandardLibrary.qll

```diff
@@ -149,6 +149,19 @@ class StringReplaceCall extends DataFlow::MethodCallNode {
       pr.flowsTo(replacer.getAReturn()) and
       map.hasPropertyWrite(old, any(DataFlow::Node repl | repl.getStringValue() = new))
     )
+    or
+    exists(
```

Comment:
Could you add a comment with a quick example of the syntax we're trying to match here?

For example
```js
    // str.replace(regex, match => {
    //   if (match === 'old') return 'new';
    // })
```